mmcv/runner/fp16_utils.py

```diff
@@ -24,6 +24,15 @@
 def cast_tensor_type(inputs, src_type, dst_type):
     """Recursively convert Tensor in inputs from src_type to dst_type.
 
+    Note:
+        In v1.4.4 and later, ``cast_tersor_type`` will only convert the
+        torch.Tensor which is consistent with ``src_type`` to the ``dis_type``.
+        Before v1.4.4, it ignores the ``src_type`` argument, leading to some
+        potential problems. For example, invoking
```

Comment:
For example,  ``cast_tensor_type(inputs, torch.float, torch.half)`` will convert all tensors in inputs to ``torch.half`` including those originally in ``torch.Int`` or other types, which is not expected.